scripts/userstyles.yml

```diff
@@ -468,18 +468,9 @@ userstyles:
     color: lavender
     readme:
       app-link: "https://github.com/mastodon/mastodon"
-      usage: |+
-        This userstyle is applied by default on a few commonly used Mastodon servers, listed at the top:
-        ```css
-        @-moz-document domain("mastodon.social"), domain("mstdn.social"), domain("fosstodon.org"), domain("piaille.fr"), domain("tech.lgbt") {
-        ```
-        If the theme isn't being applied to your instance, go to the userstyle's **Settings > Custom included sites** and add your instance to the list, for instance, `*://floss.social/*`.
       faq:
         - question: "**The theme does not look the same as the preview?**"
           answer: "Your Mastodon instance may be using its own custom CSS, which is changing the look of the theme."
-        - question: "**The theme is not working?**"
-          answer: |
-            One solution might be changing the theme to either light or dark via **Preferences** > **Appearance** > **Site theme**.
```

Comment:
I don't believe this should be removed?

Reply:
see https://github.com/catppuccin/userstyles/pull/727